src/pages/document/[id].tsx

```diff
@@ -313,6 +313,8 @@ const FamilyPage: InferGetServerSidePropsType<typeof getServerSideProps> = ({ pa
 export default FamilyPage;
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
+  context.res.setHeader("Cache-Control", "public, max-age=3600, immutable");
```

Comment:
Would it also be a good idea to set this in `ApiClient` 's `get()` function too?